src/search/pruning/stubborn_sets.h

```diff
@@ -1,29 +1,15 @@
 #ifndef PRUNING_STUBBORN_SETS_H
 #define PRUNING_STUBBORN_SETS_H
 
-#include "../abstract_task.h"
 #include "../pruning_method.h"
 
-#include "../utils/timer.h"
-
-namespace options {
-class OptionParser;
-}
+#include "../task_proxy.h"
```

Comment:
Shouldn't have a blank line between the includes. The previous blank line was because of the different subdirectories involved.